### DISCUSSION TITLE
Implement multi-criteria impacts

### DISCUSSION THREAD

**Discussion by da-ekchajzer** [electricitymaps/electricitymaps-contrib#4559] [Ideas]
Thank you for your work. Hope this issue isn't redundant with an existing one.

## Problem

The impacts related to electricity consumption are not limited to the green gas emissions. Indeed, other environmental impacts should be reported by electricity map in order to understand more broadly the effect induce by electrical consumption and avoid pollution transfer from one impact to another. 

## Solution

In a LCA (Life cycle assessment) principle, the impact of a product or service should be evaluated on the all life cycle for different impacts criteria, Here is a list of impacts criteria given in [PEF](https://ec.europa.eu/environment/eussd/smgp/ef_pilots.htm) (Produce Environmental Footprint) that could be integrated

![Screenshot_20220830_153605](https://user-images.githubusercontent.com/24867893/187451366-64add888-7197-44ca-9358-19e3f273b221.png)

see : https://ec.europa.eu/environment/eussd/pdf/footprint/PEF%20methodology%20final%20draft.pdf#page=28

## Implementation

I assume that other impacts could be calculated the same way gwp footprint is measured - with the according impact factors.

## Finding open impacts factors

If you find this issue interesting, I could look into it further.

**Comment by VIKTORVAV99** [comment]
Hmm this is a big topic and might be more suitable for a discussion instead of an issue, would you mind if I converted it into a discussion?

As regards to the question/issue itself there are certainly more data that would be useful the display  but there are a few things that makes this difficult to do at the scale Electricity Maps use:

- First of all we would have to find a good way to display said data and build it, this would take some time but might be relatively easy given time.
- We would have to gather this data and come up with a good way to model it for a whole zone or country as the data is more plant and location specific than co2 equivalents (there are some more general ones like particulate matter and ozone depletion that could be modeled just like co2eq).
This is probably the most difficult one since we would have to find data for at least every type of source in all countries to get an accurate model since most of these depend on the local environment and infrastructure to such a high degree.
- The backend would have to be modified to handle and calculate these new values (we might have to flow trace these as well).

With all that said I would personally very much like to see at least particulate matter myself.

**Comment by da-ekchajzer** [comment]
Thank you for your response.

Of course, go ahead, transfer the issue into a discussion, 

I don't know the implementation of the calculation engine, but I imagine that changing the impact format from a scalar to a vector must have quite a few architectural consequences.

Methodologically, I think that impact criteria that are linear to the electricity production should follow the same methodology mentioned here: https://github.com/electricitymaps/electricitymaps-contrib/wiki/Carbon-intensity-calculations

Beyond particulate matter, I think that other criteria are important to take into account. I tried to identify which criteria could be of interest in the context of energy production. I also mentioned if the criteria must be evaluated from specific plant characteristics. Pardon me if I am wrong, I am neither a LCA nor energy expert.

| EF Impact Category                        | Intresting for           | Plant specific | EF Impact Assessment Model                                                                                  | EF Impact Category indicators                        | Source                                          |
|-------------------------------------------|--------------------------|----------------|-------------------------------------------------------------------------------------------------------------|------------------------------------------------------|-------------------------------------------------|
| Climate Change                            | Fossil                   | No             | Bern model - Global Warming Potentials (GWP) over a 100 year time horizon.                                  | kg CO2 equivalent                                    | Intergovernmental Panel on Climate Change, 2007 |
| Ozone Depletion                           | ?                        | No             | EDIP model based on the ODPs of the World Meteorological Organization ([MMO) over an infinite time horizon. | kg CFC-11 equivalent                                 | WMO, 1999                                       |
| Ecotoxicity for aquatic fresh water       | ?                        | ?              | USEtox model                                                                                                | CTUe (Comparative Toxic Unit for ecosystems)         | Rosenbaum et al., 2008                          |
| Human Toxicity - cancer effects           | ?                        | ?              | USEtox model                                                                                                | CTUh (Comparative Toxic Unit for humans)             | Rosenbaum et al., 2008                          |
| Human Toxicity - non-cancer effects       | ?                        | ?              | USEtox model                                                                                                | CTUh (Comparative Toxic Unit for humans)             | Rosenbaum et al., 2008                          |
| Particulate Matter/Respiratory Inorganics | Fossil                   | No             | RiskPoll model                                                                                              | kg PM2.5 equivalent                                  | Humbert, 2009                                   |
| lonising Radiation - human health effects | Nuclear, coal         | No             | Human Health effect model                                                                                   | kg U215 equivalent (to air)                          | Dreicer et al., 1995                            |
| Photochemical Ozone Formation             | ?                        | ?              | LOTOS-EUROS model                                                                                           | kg NMVOC equivalent                                  | Van Zelm et al., 2008 as applied in ReCiPe      |
| Acidification                             | ?                        | ?              | Accumulated Exceedance model                                                                                | mol H+ eq                                            | Seppälä et al., ,2006; Posch et al., 2008       |
| Eutrophication - terrestrial              | ?                        | ?              | Accumulated Exceedance model                                                                                | mol N eq                                             | Seppälä et al., 2006; Posch et al., 2008        |
| Eutrophication - aquatic                  | ?                        | ?              | EUTREND model                                                                                               | fresh water: kg P equivalent marine: kg N equivalent | Struijs et al., 2009 as implemented in ReCiPe   |
| Resource Depletion - water                |  All                     | ?              | Swiss Ecoscarcity model                                                                                     | m3 water use related to local scarcity of water      | Frischknecht et al., 2008                       |
| Resource Depletion - mineral, fossil      | Renewable energy, fossil | No             | CML2002 model                                                                                               | kg antimony (Sb) equivalent                          | van Oers et al., 2002                           |
| Land Transformation                       | All                      | yes            | Soil Organic Matter (SOM) model                                                                             | Kg (deficit)                                         | Milà i Canals et al., 2007                      |

**Reply by VIKTORVAV99** [reply]
Nice table!
The only thing I want to add is that ionizing radiation affects more than nuclear (mostly coal) and because it also includes coal plants it become more plant specific so it might be hard to implement.

**Reply by da-ekchajzer** [reply]
I have added coal as you mentioned it. 
I would be interested to know why ionizing radiation induced by coal is plant specific ?

**Reply by VIKTORVAV99** [reply]
It not as much "plant" specific at it is specific to the type of coal the plant use which can differ between plants in one zone or region while still being reported together. So it has a local variance between plants cause they use fuel/coal from different places.
